index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+
+    <title>ToDo List</title>
+    <meta name="description" content="Calories application" />
+    <meta name="author" content="Bartlomiej Zdybel" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, maximum-scale=1"
+    />
+    <link rel="stylesheet" type="text/css" href="./index.css" />
+
+    <link rel="icon" href="icon_code.jpg" />
+  </head>
+
+  <body>
+      <div class="container">
+      <form class="task-form task-form__add">
```

Comment:
Firstly it would be only a block - `task-form` but in the next version of my application I wanted to create 2 similar blocks (picture below). These block have different heigh in compare to main block, so i want to make Modifier but I don't has a Element. In sum this `task-form__add` in my mind is a modifier. (hope you are understanding)
![aa](https://user-images.githubusercontent.com/29902660/54597611-240d0300-4a37-11e9-9227-6bfefc605080.png)